os-autoinst-obs-auto-submit

```diff
@@ -126,32 +126,33 @@ TMPDIR=${TMPDIR:-$(mktemp -d)}
 trap 'rm -rf "$TMPDIR"' EXIT
 
 (cd "$TMPDIR"
-for package in os-autoinst openQA os-autoinst-distri-opensuse-deps; do
-    $osc service wait "$src_project"/$package
-    $osc co -S "$src_project"/$package
+auto_submit_packages="$(osc ls "$dst_project" | grep -v '\-test$')"
```

Comment:
Note that quotes are generally not needed on assignments, e.g. the following would also work if the command outputs spaces:
```suggestion
auto_submit_packages=$(osc ls "$dst_project" | grep -v '\-test$')
```

Reply:
I am adding quotes where shellcheck demands them not anywhere else

Reply:
ok, I was wrong. shellcheck does not demand these. I changed that